plow/driver.flow

```diff
@@ -192,7 +192,9 @@ compilePlow(cache : PlowCache, file : string) -> BModule {
 
 		// OK, parse all the dependent files
 		imports = getDImports(p);
-		mods = concurrent(false, map(imports, \i -> {
+		// TODO: fix this so that concurrent running of imports compilation 
+		// wouldn't hang
```

Comment:
Do we know how it can happen?

Reply:
As I understand the reason why plow hangs here is that this place is called recursively, and at some point all concurrent workers (which number is fixed, by default it is 16) are already busy, so no new compilation jobs can be started, but since there are pending compilation jobs - this is a deadlock.